StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkSignupPane/LinkEmailElement.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  LinkEmailElement.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 2/21/23.
+//
+
+@_spi(STP) import StripeUICore
+import UIKit
+
+class LinkEmailElement: Element {
```

Comment:
This is a ~direct copy of `LinkEmailElement` from Payments. Will work with @bmelts-stripe on how we reuse this (or not)

(I've discussed in DM's but there's some complications on the fact that this email element is a bit different than Financial Connections designs)